list-filtering/solution.js

```diff
@@ -0,0 +1 @@
+filter_list = (l) => l.filter(item => typeof(item) != "string");
```

Comment:
Best practice is to use strict comparisons: 
```
!== instead of !=
```

Another way you could do it is to compare if item is equal to 'number'.
```
item => typeof(item) === 'number'
```